webpack/components/extensions/HostDetails/Tabs/PackageInstallModal.js

```diff
@@ -0,0 +1,236 @@
+import React, { useState } from 'react';
+import { Modal, Button, Dropdown, DropdownItem, DropdownToggle, DropdownDirection, DropdownToggleAction } from '@patternfly/react-core';
+import { CaretDownIcon, CaretUpIcon } from '@patternfly/react-icons';
+import { useSelector, useDispatch } from 'react-redux';
+import { FormattedMessage } from 'react-intl';
+import { Thead, Th, Tbody, Tr, Td, TableVariant } from '@patternfly/react-table';
+import { noop } from 'foremanReact/common/helpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import PropTypes from 'prop-types';
+import TableWrapper from '../../../Table/TableWrapper';
+import { useBulkSelect } from '../../../Table/TableHooks';
+import { HOST_YUM_INSTALLABLE_PACKAGES_KEY } from '../YumInstallablePackages/YumInstallablePackagesConstants';
+import { selectHostYumInstallablePackagesStatus } from '../YumInstallablePackages/YumInstallablePackagesSelectors';
+import { getHostYumInstallablePackages } from '../YumInstallablePackages/YumInstallablePackagesActions';
+import './PackageInstallModal.scss';
+import { installPackageBySearch } from './RemoteExecutionActions';
+import { katelloPackageInstallBySearchUrl } from './customizedRexUrlHelpers';
+import hostIdNotReady from '../HostDetailsActions';
+
+const InstallDropdown = ({ isDisabled, installViaRex, bulkCustomizedRexUrl }) => {
+  const [isActionOpen, setIsActionOpen] = useState(false);
+  const onActionSelect = () => {
+    setIsActionOpen(false);
+  };
+  const onActionToggle = () => {
+    setIsActionOpen(prev => !prev);
+  };
+
+  const dropdownItems = [
+    <DropdownItem key="install-rex" component="button" onClick={installViaRex}>
+      {__('Install via remote execution')}
+    </DropdownItem>,
+    <DropdownItem
+      key="install-customized-rex"
+      component="a"
+      href={bulkCustomizedRexUrl}
+      onClick={onActionSelect}
+    >
+      {__('Install via customized remote execution')}
+    </DropdownItem>,
+  ];
+  return (
+    <Dropdown
+      direction={DropdownDirection.up}
+      onSelect={onActionSelect}
+      toggle={
+        <DropdownToggle
+          toggleVariant="primary"
```

Comment:
This is documented in Patternfly but doesn't seem to work here. I think our CSS is overriding it somewhere.